pytket/extensions/azure/backends/azure.py

```diff
@@ -128,8 +128,11 @@ def process_circuits(
         - option_params: a dictionary with string keys and arbitrary values;
           key-value pairs in the dictionary are passed as input parameters to
           the backend. Their semantics are backend-dependent.
+        - job_name_prefix: A custom prefix to add to all job_names. The default is
+          to use the prefix `azure-quantum-job`.
         """
         option_params = kwargs.get("option_params")
+        job_name_prefix = kwargs.get("job_name_prefix", "job")
```

Comment:
The default value doesn't match what is stated in the docstring. Did you want to change this to "azure-quantum-job"? (The "azure-quantum" part seems redundant to me, but happy to go with that if you think it's helpful.)

Reply:
Thanks. I can add back. I changed to azure-quantum-job, but then changed back to job but forgot to update the string.